resource/config.go

```diff
@@ -372,3 +371,14 @@ func TransformAttributeMap[T any](attributes utils.AttributeMap) (T, error) {
 	}
 	return out, nil
 }
+
+// NewConfigValidationError returns a config validation error occurring at a given path.
+func NewConfigValidationError(path string, err error) error {
+	return fmt.Errorf("Error validating. Path: %q Error: %w", path, err)
```

Comment:
I'm surprised linter is letting you (it might not be letting you) capitalize these errors messages.